content/shmem_barrier.tex

```diff
@@ -38,7 +38,7 @@
 \apidescription{
     \FUNC{shmem\_barrier} is a collective synchronization routine over an
     \activeset.  Control returns from \FUNC{shmem\_barrier} after all \acp{PE} in
-    the \activeset (specified by \VAR{PE\_start}, \VAR{logPE\_stride}, and
+    the \activeset(specified by \VAR{PE\_start}, \VAR{logPE\_stride}, and
```

Comment:
Looking at the implementation of the ```\activeset``` macro, it already contains an ```\xspace```, so this change should not be necessary (i.e. it should have no effect).